livre/eq/Config.cpp

```diff
@@ -229,11 +229,8 @@ bool Config::frame()
         const uint32_t start = frameMin;
         const uint32_t end = std::max( frameMax, current );
         const int32_t delta = params.animation;
+        const uint32_t interval = end - start;
```

Comment:
I don't know what you mean with the limit...
Anyways, I'll try to add a test in this commit to finish this madness once and for all.